modules/nw-metallb-bgppeer-cr.adoc

```diff
@@ -70,6 +70,10 @@ If you specify this field, you must specify the same value in every BGP peer cus
 |`string`
 |Optional: Specifies the MD5 password to send to the peer for routers that enforce TCP MD5 authenticated BGP sessions.
 
+|`spec.passwordSecret`
```

Comment:
I'd mention the fact that the secret must live in the metallb's namespace and be of type basic-auth